src/python/pants/backend/python/dependency_inference/parse_python_imports_test.py

```diff
@@ -42,7 +42,7 @@ def assert_imports_parsed(
     rule_runner: RuleRunner,
     content: str,
     *,
-    expected: dict[str, int],
+    expected: dict[str, tuple[int, bool]],
```

Comment:
It's not very clear what the `bool` refers to. Some ideas to make this more clear:

* Expect a `ParsedPythonImportInfo`, but have it default to `weak=False`.
* Use an `enum` rather than `bool` to track `weak`, as mentioned above.
* Use constants like `WEAK = True` and `STRONG = False`.